src/components/30-organisms/modal/index.scss

```diff
@@ -0,0 +1,83 @@
+html,
+body {
+  width: 100%;
+  height: 100%;
+}
+
+.o-modal {
+  display: none;
+  width: 100%;
+  height: 100%;
+
+  position: fixed;
+  z-index: 1;
```

Comment:
Hm.... This would not work well on axa.ch.

We should discuss this in an after daily, as this is a serious problem in webhub generally.